examples/react/src/containers/AnimalForestChat.js

```diff
@@ -17,8 +17,10 @@ export default class extends Component {
     const randomUser = this.getRandomUser();
     this.uuid = randomUser.uuid;
     this.designation = randomUser.designation;
-    this.userName = randomUser.firstName + ' ' +  randomUser.lastName;
-    this.userProfileImage = randomUser.profileImage.lgImage;
+    this.userProfile = {
+      name: randomUser.firstName + ' ' +  randomUser.lastName,
+      image: randomUser.profileImage.lgImage
```

Comment:
nitpick: Add trailing comma @ ```image: randomUser.profileImage.lgImage,```

Reply:
Seems that trailing commas are used in some objs but not all. Would we be fine with updating or adding a lint rule to do so? @polarweasel